helpers/dragRangeSliderLowerBoundTo.ts

```diff
@@ -5,11 +5,13 @@ declare namespace WebdriverIOAsync {
 }
 
 browser.addCommand('dragRangeSliderLowerBoundTo', async (value: number) => {
-  const slider = await browser.$('.rheostat-horizontal');
-  let lowerHandle = await browser.$('.ais-RangeSlider .rheostat-handle-lower');
-  const upperHandle = await browser.$(
-    '.ais-RangeSlider .rheostat-handle-upper'
+  const slider = await browser.$('.slider-rail, .rheostat-background');
```

Comment:
so is the code here applicable for both the dom of React InstantSearch & InstantSearch because of this comma selector?

Reply:
Yes, it will return the first element it find with this selector.

This is equivalent of doing a `document.querySelector('.slider-rail, .rheostat-background')`